### PR TITLE
4973 add ids counts

### PR DESCRIPTION
**Pull request by chrisakinrinade**
**Explain the details for making this change. What existing problem does the pull request solve?**
Adding IDS Counts component to the web component repo. Currently supports hyperlinked, compact, and disabled counts.

**Related github/jira issue (required)**:
Fixes https://github.com/infor-design/enterprise/issues/4973

**Steps necessary to review your pull request (required)**:
1. Pull this branch, build, and run the app
2. Go to http://localhost:4300/ids-counts
3. Check the code implementations and functional tests.

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — COMMENTED** [review]
Looking really good now. A few things tho, please:

1. few comments on the DOCS and wizard change
1. https://github.com/infor-design/enterprise-wc/pull/114/checks?check_run_id=2559469824#step:7:10 some lint issues (can run `npm run lint` locally to test) and can also setup your linter in VS code if not setup.
1. Example looks good but only one problem: the "actionable" links dont seem to be azure anymore.

File: test/ids-wizard/ids-wizard-func-test.js [hunk 1]
```diff
@@ -52,114 +52,114 @@ describe('IdsWizard Tests', () => {
...
-    elem.clickable = true;
-
-    const marker = elem.shadowRoot.querySelector(
-      `.step[step-number="${stepNumber}"] .step-marker`
-    );
-    marker.click();
-
-    expect(elem.stepNumber).toEqual(2);
-  });
+  // it('renders correctly', () => {
```
[the remark this answers is not in the document]
Comment (chrisakinrinade) [reply test/ids-wizard/ids-wizard-func-test.js:55]: Yup, twas a mistake

**Comment by tmcconechy** [comment]
I think this is getting even closer. But can you also 

a) What you can do copy the files https://github.com/infor-design/enterprise/blob/main/package.json and save over so its net unchanged.

**Review by tmcconechy — APPROVED** [review]
Looks good now 👍🏻  Assuming all the tests pass :)

**Review by tmcconechy — APPROVED** [review]

**Review by EdwardCoyle — APPROVED** [review]
Looks really good @chrisakinrinade, thank you! 🍌

**Pull request merged**